### PR TITLE
[Explorer] Creates TransactionType Component

### PR DESCRIPTION
**Pull request by Andrew47**
This creates a TransactionType Component following APPS-31.

image

image

### CODE REVIEW

**Review by Jordan-Mysten — CHANGES_REQUESTED** [review]

File: apps/explorer/src/components/transaction-card/TxCardUtils.tsx [hunk 1]
```diff
@@ -101,22 +100,10 @@ export function TxAddresses({ content }: { content: LinkObj[] }) {
 }
 
 function TxStatusType({ content }: { content: TxStatus }) {
```
Comment (pchrysochoidis) [inline apps/explorer/src/components/transaction-card/TxCardUtils.tsx:102]: Is this wrapper component now necessary? Why not just remove it and use the new component instead?

File: apps/explorer/src/ui/TxType.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+export type TxTypeProps = {
+    isSuccess?: boolean;
+    count?: string;
+    children: ReactNode;
+};
+
+export function TxType({ isSuccess, count, children }: TxTypeProps) {
+    return (
+        <div className="flex flex-col items-start">
```
[the remark this answers is not in the document]
Comment (Andrew47) [reply apps/explorer/src/ui/TxType.tsx:17]: If I do that I get this
image

File: apps/explorer/src/ui/TransactionType.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+};
+
+export function TransactionType({
+    isSuccess,
+    count,
+    type,
+}: TransactionTypeProps) {
+    return (
+        <div className="flex flex-col items-start">
+            <div className="flex items-center justify-center gap-1.5">
```
Comment (pchrysochoidis) [inline apps/explorer/src/ui/TransactionType.tsx:24]: I think if you remove `justify-center` from here you can get rid the other div

**Review by pchrysochoidis — COMMENTED** [review]
Just a small thing (related to a comment from Jordan), rather than that looks good to me

File: apps/explorer/src/ui/TransactionType.tsx (hunk 3 above)
Comment (Andrew47) [reply apps/explorer/src/ui/TransactionType.tsx]: Excellent - I tried this and it looks like this works as a refactoring.

**Review by pchrysochoidis — APPROVED** [review]

**Pull request merged**